elastalert/kibana.py

```diff
@@ -166,6 +166,8 @@
                   u'style': u'dark',
                   u'title': u'ElastAlert Alert Dashboard'}
 
+kibana4_time_temp = "(refreshInterval:(display:Off,section:0,value:0),time:(from:'%s',mode:absolute,to:'%s'))"
```

Comment:
Can we either set a non-zero refresh time or allow this to be customized? Would be pretty easy to have this templated by field name, then splat in a dict straight from the YAML config of whatever vars the user wanted to add.

Too loosely typed maybe?